### ISSUE TITLE
Picture added event

### ISSUE THREAD

**Issue by damjan-dimitrioski** [Tapfiliate/tapatar#10]
How to know when a picture was added ?

I'm configuring a jquery validator for the tapatar field, but was not able to find a proper way to recognize when value was changed.

How can the validation be accomplished ?

Thanks.

**Comment by thomask** [comment]
Fixed by a0fedf774b94a5cf49ca8f6044974bb917ff7e08

You can now listen for the `tapatar.source.image_data.set` and `tapatar.source.picked` events.

```
$('#tapatar').tapatar({
    sources: {
      gravatar: {
        email: 'foo@bar.com'
      },
      facebook: {
        appId: 1234
      }
    }
}).on('tapatar.source.image_data.set', function(ev, source){
  // handle
}).on('tapatar.source.picked', function(ev, source){
  // handle
});
```

**Comment by damjan-dimitrioski** [comment]
Thanks a lot :), I really appreciate the patch.
